tenable/io/v3/base/endpoints/explore.py

```diff
@@ -77,29 +117,24 @@ def search(
                 If ``return_json`` was set to ``True``, then a response
                 object is instead returned instead of an iterable.
 
-        Examples:
-
-            >>>
         '''
-        kwargs['fields'] = fields
+        sort = kwargs.get('sort')
+        next_token = kwargs.get('next')
+        sort = generate_sort_data(sort, is_sort_with_prop)
         kwargs['sort'] = sort
-        kwargs['filter'] = filter
-        kwargs['limit'] = limit
-        kwargs['next'] = next
-        if not schema_cls:
-            schema_cls = SearchSchema
-        if not iterator_cls:
-            # todo - commenting this temporarily
-            # iterator_cls = ExploreSearchIterator
-            pass
-        schema = schema_cls()
-        payload = schema.dump(schema.load(kwargs))
+        if not next_token:
+            schema = schema_cls(unknown=EXCLUDE)
```

Comment:
This usage should be discouraged as it allows for invalid attributes to be silently dropped.